.github/workflows/CIMaster.yml

```diff
@@ -0,0 +1,92 @@
+name: CI
+# Run on master
+on:
+  schedule:
+    - cron: '0 2 * * *'  # Daily at 2 AM UTC (8 PM CST)
+  push:
+    branches: [master]
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }}
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.version == 'nightly' }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - 1
+        os:
+          - ubuntu-latest
+          - macOS-latest
+          - windows-latest
+        arch:
+          - x64
+        include:
+          # Add a 1.0 job just to make sure we still support it
+          - os: ubuntu-latest
+            version: 1.0.5
```

Comment:
Would 1.0 here just grab the latest 1.0.x version?

Reply:
I think I read somewhere that if you do `1.0` it strips out the `.0` for some reason and grabs the latest `1.x` package. Found the link https://github.com/invenia/JLSO.jl/pull/81#discussion_r516857524